open-craft/configuration
# BB-4521 Create mongo indexes in forum and edx_platform deployment

Create mongo indexes in deployment for forum and edx_platform
---
Jira ticket: https://tasks.opencraft.com/browse/BB-4521

Test Steps:
- Deploy a new appserver in the stage server using this branch for configuration
- Once it is correctly deployed, you can follow the verification steps [here](https://docs.google.com/document/d/187IoKOSkhDwuUF0-Tja-bhTE2teAGxTwA9a0Y3h9Q7s/edit) to check if indexes have been correctly created for the edxapp mongodb and forum mongodb
    - `mongo --authenticationMechanism SCRAM-SHA-1 -u <username> -p <password> --host rs0/mongodb.opencraft.hosting,mongodb-secondary-1.net.opencraft.hosting,mongodb-secondary-2.net.opencraft.hosting <db-name>`
    - `db.<collection-name>.getIndexes()`
    - or 
    ```
    db.getCollectionNames().forEach(function(collection) {
   indexes = db.getCollection(collection).getIndexes();
   print("Indexes for " + collection + ":");
   printjson(indexes);
   });
    ```
   to get all indexes created

Configuration Pull Request
---

Make sure that the following steps are done before merging
